python/src/DistFunc_doc.i.in

```diff
@@ -26,6 +112,131 @@ Examples
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::DistFunc::logdHypergeometric
+"Logarithm of the probability function of an hypergeometric distribution.
+
+Parameters
+----------
+n : int, :math:`n\geq 0`
+    The population size
+k : int, :math:`0\leq k\leq n`
+    The number of candidates in the population
+m : int, :math:`0\leq m\leq n`
+    The number of individuals in a draw
+x : int, :math:`x\geq 0`
+    The number of candidates in a draw
+
+Returns
+-------
+logp : float
+    The natural logarithm of the probability to get :math:`x` candidates in a draw.
+
+Notes
+-----
+This method is based on an algorithm similar to Loader's algorithm, the *fast* and *accurate* method described in [loader2000]_, with the further improvements mentioned in [dimitriadis2016]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> p = ot.DistFunc.logdHypergeometric(10, 4, 7, 2)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::dHypergeometric
+"The probability function of an hypergeometric distribution.
+
+Parameters
+----------
+n : int, :math:`n\geq 0`
+    The population size
+k : int, :math:`0\leq k\leq n`
+    The number of candidates in the population
+m : int, :math:`0\leq m\leq n`
+    The number of individuals in a draw
+x : int, :math:`x\geq 0`
+    The number of candidates in a draw
+
+Returns
+-------
+p : float
+    The probability to get :math:`x` candidates in a draw.
+
+Notes
+-----
+This method is based on an algorithm similar to Loader's algorithm, the *fast* and *accurate* method described in [loader2000]_, with the further improvements mentioned in [dimitriadis2016]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> p = ot.DistFunc.dHypergeometric(10, 4, 7, 2)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::pHypergeometric
+"The cumulative probability function of an hypergeometric distribution.
+
+Parameters
+----------
+n : int, :math:`n\geq 0`
+    The population size
+k : int, :math:`0\leq k\leq n`
+    The number of candidates in the population
+m : int, :math:`0\leq m\leq n`
+    The number of individuals in a draw
+x : int, :math:`x\geq 0`
+    The number of candidates in a draw
+tail : bool
+    Flag to tell if it is the CDF or its complement which is evaluated
+
+Returns
+-------
+p : float
+    The probability to get at most :math:`x` candidates in a draw.
+
+Notes
+-----
+This method is based on a summation of the probability function toward the upper bound or the lower bound of the range depending on the position of :math:`x` wrt the mode :math:`\left\lfoor\dfrac{(k+1)(m+1)}{n+2}\right\rfloor` of the distribution, then take the complement if needed.
```

Comment:
\lfoor -> \floor ?

Reply:
ok this time it should be good. Fixed the test, the LaTeX operator and the length of the lines in DistFunc_doc.i.in!